client.go

```diff
@@ -32,57 +30,45 @@ type Token struct {
 // Provides the client and associated elements for interacting with the
 // Tesla API
 type Client struct {
-	Auth  *Auth
-	Token *Token
-	HTTP  *http.Client
+	Auth         *Auth
+	Token        *Token
+	HTTP         *http.Client
+	BaseURL      string
+	StreamingURL string
 }
 
-var (
-	AuthURL      = "https://owner-api.teslamotors.com/oauth/token"
-	BaseURL      = "https://owner-api.teslamotors.com/api/1"
-	ActiveClient *Client
-)
+var AuthURL = "https://owner-api.teslamotors.com/oauth/token"
```

Comment:
Why is this also not a const, if the idea is to manipulate these on the Client if needed?

Reply:
`AuthURL` is out of scope for this change and it's still used in tests.  I plan to remove it as part of the oauth2 work.